add configs for wemos lolin s3 mini

tzapu opened the pull request:
works perfectly for this board
https://www.wemos.cc/en/latest/s3/s3_mini.html
using a 8 pin header and these pins 
<img width="242" alt="image" src="https://github.com/user-attachments/assets/4a35fc1c-2a29-461a-b019-04140dabbfcc" />

F1p commented:
Strange, same chip as the Atom S3 lite

gekkekoe commented:
he has a relugar atom lite, so the non S3 https://shop.m5stack.com/products/atom-lite-esp32-development-kit

tzapu commented:
i dropped those old atoms, replaced with this https://www.wemos.cc/en/latest/s3/s3_mini.html
and made work with this config, including rgb status 

thanks for merging
